.github/workflows/docker-weekly.yml

```diff
@@ -0,0 +1,24 @@
+name: weekly docker build
+
+on:
+  schedule:
+    # Monday midnight
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+
+permissions:
+  packages: write
+  contents: write
+  id-token: write
+
+jobs:
+  weekly-container-build:
+    uses: ./.github/workflows/reusable-container-workflow.yaml
+    with:
+      dockerfile: dev
```

Comment:
nit: maybe rename "dockerfile" to "flavour" but besides this LGTM

Reply:
Changed this spot to build_type, because, usually, the distro is referenced as the flavor. Now called that way as well.